packages/graphback-datasync/src/util.ts

```diff
@@ -39,3 +44,35 @@ export const DataSyncFieldNames = {
   lastUpdatedAt: '_lastUpdatedAt',
   deleted: '_deleted'
 }
+
+
+
+export interface DataSyncModelConflictConfig {
+  enabled: boolean
+  conflictResolution?: ConflictResolutionStrategy
+}
+
+export interface DataSyncModelConfigMap {
+  [modelName: string]: DataSyncModelConflictConfig
+}
+
+export interface ConflictResolutionStrategy {
+  resolveUpdate(conflict: ConflictMetadata): any
+}
+
+export const ServerSideConflictResolution: ConflictResolutionStrategy = {
+  resolveUpdate(conflict: ConflictMetadata): any {
+    const { serverDiff, clientDiff } = conflict;
+
+    const resolved: any = {};
+
+    Object.keys(clientDiff).forEach((fieldName: string) => {
```

Comment:
I cannot confirm if this algorithm works. Would be possible to explain motivation here.

Conflict detected by version doesn't mean actual conflict. Our implementation should know if we have conflict or mergability. Interface should clearly represent that.

Reply:
> I cannot confirm if this algorithm works. Would be possible to explain motivation here.

This is merging the fields that weren't changed

> Conflict detected by version doesn't mean actual conflict. Our implementation should know if we have conflict or mergability. Interface should clearly represent that.

I though this was doing it:https://github.com/aerogear/graphback/blob/94c0db84ebb75ff2bce18350da8680bd144fb27b/packages/graphback-datasync/src/providers/DataSyncConflictProvider.ts#L84-L121